src/Vehicle/VehicleServoFactGroup.cc

```diff
@@ -0,0 +1,74 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+#include <iostream>
+
+#include "VehicleServoFactGroup.h"
+#include "MAVLinkProtocol.h"
+#include "Vehicle.h"
+#include "QGCGeo.h"
+
+#define DAY_LIGHT_SERVO   3
+#define NIGHT_LIGHT_SERVO 11
+#define PWM_VALUE         1500
+
+const char* VehicleServoFactGroup::_dayLightFactName   = "dayLight";
+const char* VehicleServoFactGroup::_nightLightFactName = "nightLight";
+
+VehicleServoFactGroup::VehicleServoFactGroup(QObject* parent)
+    : FactGroup(1000, ":/json/Vehicle/ServoStatusFactGroup.json", parent)
+    , _dayLightFact   (0, _dayLightFactName,   FactMetaData::valueTypeBool)
+    , _nightLightFact (0, _nightLightFactName, FactMetaData::valueTypeBool)
+{
+    _addFact(&_dayLightFact,   _dayLightFactName);
+    _addFact(&_nightLightFact, _nightLightFactName);
+
```

Comment:
This is servo functionality status, correct?

Reply:
this is fact group classes, it part of logic of handling events from mavlink and updating states